tmt/steps/provision/connect.py

```diff
@@ -108,6 +109,13 @@ def go(self):
             self.debug('Using private key authentication.')
             data['key'] = key
 
+        if ssh_option:
+            self.info('ssh options', ssh_option, 'green')
```

Comment:
Should be formatted to be more user friendly. `run --ssh-option StrictHostKeyChecking=no` prints 
`ssh options: ('StrictHostKeyChecking=no',)`

Also could it be printed under same name as  cli option (`--ssh-option`)?

Reply:
Hi. Good point. However, there can be more --ssh-option on cli.
--ssh-option blah=1 --ssh-option kuku=2

Maybe it can by printed line by line?

ssh-option: blah=1
ssh-option: kuku=2
...

Line by line approach seems to be nice from my pow.

Reply:
Problem is that values aren't ready for copy&paste if there are multiple values (comma separated, some values can be omitted and last value has "and" before it)
So IMHO pointing user to the correct option is enough.